pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -129,6 +129,9 @@ func (rc *reconciler) updateSyncTime() {
 	rc.timeOfLastSync = time.Now()
 }
 
+// This path seems to be issuing bulk-verify, but, the csi- logic is verifying one at a time?
```

Comment:
iic, sequential processing after bulk verify is the behaviour as mentioned here https://github.com/kubernetes/kubernetes/issues/84169#issuecomment-545692734

Reply:
Ack, thanks.